### PR TITLE
Backend refactor

### PR DESCRIPTION
**Pull request by DumboOctopus**
TL;DR: put routes in separate file, made controllers, made stylus build to `public/css` instead of `static/css`
Directory structure:
```
routes/
controllers/
source/
 -templates/
 -stylesheets/
public/
 - js/
 - css/
 - images/
```
The `routes` folder holds all the files that specify the routes. As of now, we only have one file (`static_pages_routers.js`) which handles all the pages we have so far. Having a folder for routes is kinda extra as of now but later if we want to make the website more complicated, it'll save us the step of refactoring again.

The routes link to a function in a controller, which are all in the controllers folder. Since we only have one route file, we have only one controller file (`static_pages_controller.js`). This is also kinda extra for now but later on it will save us some effort.

Lastly, the `npm run build-css` now compiles the stylus files to `public/css`

### CODE REVIEW

**Comment by DumboOctopus** [comment]
@amikhalev, that's true.. it should be part of the gitignore.
